client/js/widgets/pile.js

```diff
@@ -90,34 +109,105 @@ class Pile extends Widget {
         batchStart();
         for(const c of this.children())
           await c.set('z', Math.floor(Math.random()*10000));
         showOverlay();
         batchEnd();
       });
-      $('#pileOverlay').appendChild(shuffleButton);
+      buttonBar2.appendChild(shuffleButton);
+
+
+
+
+      const splitLabel = document.createElement('div');
+      splitLabel.className = 'inputtext optiontext';
+      const thisField = { type: 'text',text:'-OR-' };
+      formField(thisField, splitLabel, 'pileTitle')
+      $('#pileOverlay > .modal').appendChild(splitLabel);
 
       const childCount = this.children().length;
       const countDiv = document.createElement('div');
-      countDiv.textContent = `/ ${childCount}`;
-      $('#pileOverlay').appendChild(countDiv);
+      countDiv.className = 'countInput';
+
+      $('#pileOverlay > .modal').appendChild(countDiv);
+
       const splitInput = document.createElement('input');
       splitInput.type = 'number';
       splitInput.value = Math.floor(childCount/2);
-      splitInput.min = 0;
-      splitInput.max = childCount;
-      countDiv.prepend(splitInput);
-      const splitLabel = document.createElement('label');
-      splitLabel.textContent = 'Split: ';
-      countDiv.prepend(splitLabel);
+      splitInput.min = 1;
+      splitInput.max = childCount - 1;
+      splitInput.setAttribute("aria-valuemin", 0);
+      splitInput.setAttribute("aria-valuemax", childCount);
+      splitInput.setAttribute("aria-valuenow", splitInput.value);
+      splitInput.addEventListener('input', async e=>{
+        if(splitInput.value > (childCount - 1)){
+          splitInput.value = childCount - 1;
+        }
+        if(splitInput.value < 1){
+          splitInput.value = 1;
+        }
+        denominatorInput.value = childCount - splitInput.value;
+        splitInputSlider.value = childCount - denominatorInput.value;
+      });
+      countDiv.appendChild(splitInput);
+
+      const splitInputSlider = document.createElement('input');
+      splitInputSlider.type = 'range';
+      splitInputSlider.value = Math.floor(childCount/2);
+      splitInputSlider.min = 1;
+      splitInputSlider.max = childCount - 1;
+      splitInputSlider.setAttribute("aria-valuemin", 0);
+      splitInputSlider.setAttribute("aria-valuemax", childCount);
+      splitInputSlider.setAttribute("aria-valuenow", splitInputSlider.value);
+      splitInputSlider.addEventListener('input', async e=>{
+        splitInput.value = splitInputSlider.value;
+        denominatorInput.value = childCount - splitInput.value;
+      });
+      countDiv.appendChild(splitInputSlider);
+
+      const denominatorInput = document.createElement('input');
+      denominatorInput.value = childCount - splitInput.value;
+      denominatorInput.type = 'number';
+      denominatorInput.min = 1;
+      denominatorInput.max = childCount - 1;
+      denominatorInput.setAttribute("aria-valuemin", 0);
+      denominatorInput.setAttribute("aria-valuemax", childCount);
```

Comment:
Same question. Does `aria` support even make sense? I'm pretty sure that games will never be playable using a screen reader. But I don't know if it's useful for any other circumstances.